impexp-config/src/main/resources/org/citydb/config/i18n/language_de.properties

```diff
@@ -507,7 +507,9 @@ pref.kmlexport.label.deactivateKmz=KMZ deaktivieren
 pref.kmlexport.label.deactivateGlTF=glTF deaktivieren
 pref.kmlexport.label.kmzGltfWarning=Bei Verwendung des glTF-Formats kann nicht das KMZ-Format verwendet werden
 pref.kmlexport.label.notCreateColladaFiles=COLLADA-Dateien (.dae) nicht erzeugen
-pref.kmlexport.label.embedTexturesInGltfFiles=Texturen in glTF-Dateien (.gltf) einbetten
+pref.kmlexport.label.embedTexturesInGltfFiles=Texturen in glTF- bzw. Binary-glTF-Dateien (.glTF bzw. .glb) einbetten
+pref.kmlexport.label.exportGltfBinary=Binary-glTF- (.glb) anstatt glTF-Dateien (.glTF) exportieren
+pref.kmlexport.label.enableGltfDracoCompression=Geometrien mithilfe von Draco-Kompression exportieren
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).